lib/internal/test_runner/runner.js

```diff
@@ -808,6 +809,6 @@ function run(options = kEmptyObject) {
 }
 
 module.exports = {
-  FileTest, // Exported for tests only
+  FileTest,
```

Comment:
Why was this comment changed?

Reply:
Whoops. I was doing `instanceof FileTest` in the old implementation within the `getFiles`, so it was not for tests only anymore. Going to revert that.